tests/FSharp.Test.Utilities/Assert.fs

```diff
@@ -1,49 +1,46 @@
 namespace FSharp.Test
 
 module Assert =
-    open FluentAssertions
     open System.Collections
     open Xunit
     open System.IO
 
+    // Equivalent, with message
     let inline shouldBeEqualWith (expected : ^T) (message: string) (actual: ^U) =
-        actual.Should().BeEquivalentTo(expected, message) |> ignore
+        try
+            Assert.Equivalent(expected, actual)
+        with e ->
+            Assert.True(false, message);
+
+    // Equivalent, with no message
+    let inline shouldBe (expected : ^T) (actual : ^U) =
```

Comment:
I'd keep the name:
```suggestion
    let inline shouldBeEquivalentTo (expected : ^T) (actual : ^U) =
```

`shouldBe` used to carry "Equal" semantics, not "Equivalent", we should keep it.

Reply:
You may not have noticed, but the types to the Should be may may be different, ^T and ^U.  I guess that under the cover FluentAssertions does the necessary comparison.  XUnit doesn't have an equivalent overload.

I can try by down casting to object and running the tests, which may work, although my intuition is that it probably won't.  It depends how much the test cases use shouldBe with different types.

Reply:
The downcasting for the shouldBe implementation appears to work fine.  So we can go with it.  Good eyes.